web/yo/app/scripts/controllers/gene.js

```diff
@@ -1311,16 +1383,28 @@ angular.module('oncokbApp')
                     ReviewResource.accepted.push(uuid.getText());
                 }
             }
+            function acceptItemFire(arr, uuid) {
+                _.each(arr, function(item) {
+                    if (mainUtils.needReviewFire(item.uuid)) {
+                        delete item.reviewObj.lastReviewed;
+                        delete $rootScope.metaFire[item.uuid];
+                        ReviewResource.accepted.push(item.uuid);
+                    }
+                });
+                if (uuid) {
+                    ReviewResource.accepted.push(uuid);
+                }
+            }
             $scope.modelUpdate = function(type, mutation, tumor, ti, treatment) {
                 switch (type) {
                 case 'GENE_SUMMARY':
-                    acceptItem([{reviewObj: $scope.gene.summary_review, uuid: $scope.gene.summary_uuid}], $scope.gene.summary_uuid);
+                    acceptItemFire([{reviewObj: $scope.geneFire.summary_review, uuid: $scope.geneFire.summary_uuid}], $scope.geneFire.summary_uuid);
```

Comment:
I think we can also stick with original naming, make less changes.

Reply:
This will be taken care of in the Review related refactor.